apollo-router/src/uplink/license_enforcement.rs

```diff
@@ -178,7 +178,7 @@ impl LicenseEnforcementReport {
                 .name("Operation aliases limiting")
                 .build(),
             ConfigurationRestriction::builder()
-                .path("$.preview_persisted_queries")
+                .path("$.persisted_queries")
```

Comment:
Similarly want to double-check my understanding that this expression is only ever applied to a YAML file that has been "migrated".

Reply:
Yes, both license enforcement and metrics are applied to `Configuration::validated_yaml` which has already had migrations applied.